src/install/assets/install.html.twig

```diff
@@ -1,396 +1,290 @@
+                        <tr>
+                            <td>PHP safe mode</td>
+                            <td class="red">ON</td>
+                            <td>PHP safe mode should be OFF</td>
+                        </tr>
+                    {% endif %}
+                    {% for ext, loaded in extensions %}
+                        <tr>
+                            <td>PHP extension: <strong>{{ ext }}</strong></td>
+                            <td class="{{ loaded ? 'green' : 'red'}}">{{ loaded ? 'OK' : 'FAIL'}}</td>
+                            <td>{% if not loaded %}Contact your server administrator to enable <strong>PHP {{ ext }} extension</strong>{% endif %}</td>
+                        </tr>
+                    {% endfor %}
+                    {% for file, writable in files %}
+                        {% if not writable %}
+                            <tr>
+                                <td>{{ file }}</td>
+                                <td class="red">FAIL</td>
+                                <td>Please make sure that file exists and is writable.</td>
+                            </tr>
+                        {% endif %}
+                    {% endfor %}
+                    {% for folder, writable in folders %}
+                        {% if not writable %}
+                            <tr>
+                                <td>{{ folder }}</td>
+                                <td class="red">FAIL</td>
+                                <td>Please make sure that directory exists and is writable.</td>
+                            </tr>
+                        {% endif %}
+                    {% endfor %}
+                </tbody>
+            </table>
+        </details>
+
+        {% if canInstall == false %}
+
+        <p>Meet all system requirements, then refresh the page to see the setup form.</p>
+
+        {% else %}
+            <form id='form' action="/install/install.php?a=install" method="post">
+                <details id="step-2">
+                    <summary>Step 2) Enter database details</summary>
+
+                    <p>Enter your MySQL connection information.</p>
+
+                    <fieldset>
+                        <label for="database_hostname">Hostname <span class="red">*</span></label>
+                        <input type="text" id="database_hostname" name="database_hostname" value="{{ database_hostname|default('localhost') }}" placeholder="localhost, mysql, db.domain.com, 12.34.56.78 ..." required>
+                        <small>Usually <strong>localhost</strong>. For Docker, use <strong>mysql</strong>.</small>
+    
+                        <label for="database_port">Port <span class="red">*</span></label>
+                        <input type="number" id="database_port" name="database_port" value="{{ database_port|default('3306') }}" placeholder="Usually 3306." required>
+    
+                        <label for="database_name">Name <span class="red">*</span></label>
```

Comment:
I would suggest using "Database Name" as the label here, as there is another Name field for the Admin later.

Reply:
Yep, I would agree that makes sense.